ui/src/criteria/attribute.tsx

```diff
@@ -46,29 +71,64 @@ class _ implements CriteriaPlugin<Data> {
   }
 
   generateFilter() {
-    if (this.data.selected.length === 0) {
-      return null;
-    }
+    let operands: tanagra.Filter[] | undefined;
+    if (isValid(this.data.min) && isValid(this.data.max)) {
+      operands = [
+        {
+          binaryFilter: {
+            attributeVariable: {
+              variable: "person",
```

Comment:
Is it intentional that this plugin would **only** work on the person entity and not others? It's fine if this is just not part of the current work, but probably should have a TODO here to track the current status

Reply:
Yes. this is intentionally work on "person". because the other types include in the concept.tsx